HealthSDK/GiniHealthSDK/Tests/GiniHealthSDKTests/MockHelpers/Utils.swift

```diff
@@ -7,16 +7,31 @@
 
 import UIKit
 import GiniHealthAPILibrary
```

Comment:
[Clarification]: Could you please check if we need to import GiniHealthAPILibrary?

Reply:
Unfortunatelly struct PaymentProviderResponse is only available in GiniHealthAPILibrary and for this test we need that structure.